### PR TITLE
Completed javascript-timers exercise.

### PR DESCRIPTION
**Pull request by saraysc**
timeout
interval

### CODE REVIEW

File: javascript-timers/interval.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+function timeout() {
+  $countdown.textContent -= 1;
+  if ($countdown.textContent === '0') {
+    $countdown.textContent = '~Earth Beeeeloooowwwww Us~';
+  }
+}
+
+var i = setInterval(timeout, 1000);
+setTimeout(function () { clearInterval(i); }, 4000);
```
Comment (Zhiwutian) [inline javascript-timers/interval.js:11]: you don't want to use setTimeout here, you want to call clearInterval in the function when the countdown is complete

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
update needed

File: javascript-timers/interval.js (hunk 1 above)
Comment (saraysc) [reply javascript-timers/interval.js]: timeout2

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**